save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestSuitesService.kt

```diff
@@ -114,7 +115,7 @@ class TestSuitesService(
                 testSuiteDto.testRootPath,
                 testSuiteDto.testSuiteRepoUrl,
             )
-            log.info("Mark test suite ${testSuite.name} with id ${testSuite.id} as obsolete")
```

Comment:
It's should be kind of rare and scheduled task, does this message anyway annoying?

Reply:
Ok, will change back to `info`